CGPs/cgp-0190/mainnet.json

```diff
@@ -9,10 +9,12 @@
       "value": "0"
     },
     {
-        "contract": "FeeCurrencyWhitelist",
-        "function": "addToken",
+        "contract": "FeeCurrencyDirectory",
+        "function": "SetCurrencyConfig",
         "args": [
-            "0x4f604735c1cf31399c6e711d5962b2b3e0225ad3"
+            "0x4f604735c1cf31399c6e711d5962b2b3e0225ad3",
+            "0x765DE816845861e75A25fCA122bb6898B8B1282a",
```

Comment:
this should be SortedOracle contract: `0xefB84935239dAcdecF7c5bA76d8dE40b077B7b33`